cross/rabbitmq-server/patches/001-makefile.patch

```diff
@@ -0,0 +1,11 @@
+--- Makefile.orig	2019-05-17 07:43:31.000000000 +0200
++++ Makefile	2021-02-10 14:48:23.931114873 +0100
+@@ -250,7 +250,7 @@
+ 
+ DESTDIR ?=
+ 
+-PREFIX ?= /usr/local
++PREFIX ?= /usr/local/rabbitmq-server
```

Comment:
you don't need a patch for this
if `PREFIX` is already defined the `?=` does not overwrite it.

And please use `/var/packages/rabbitmq-server/target` instead of `/usr/local/rabbitmq-server`  (the later will not be supported in DSM 7+)

I have solved this with a custom install target:
`$(RUN) DESTDIR=$(INSTALL_DIR) $(MAKE) install PREFIX=$(INSTALL_PREFIX) RMQ_ROOTDIR=$(INSTALL_PREFIX)`